### ISSUE TITLE
Trip reports show incorrect start time if the trip started before the report start time

### ISSUE THREAD

**Issue by chrissfoot** [traccar/traccar#3307]
When you do a trip report and a trip started before your specified start time it just uses the time you specified as the start time regardless of when the trip actually started. However, if the trip ended after your end time it doesn't return the trip at all but does at least seem to respect the actual end time of the trip

**Comment by tananaev** [comment]
I would need to see examples, and what you expect to see.

**Comment by chrissfoot** [comment]
Hi Tananaev,

Sorry, I should have clarified a bit! 

I've attached a couple of reports that illustrate what I think the problem is. The correct one was a report starting at midnight, showing the trip starting at 06:34. The second report shows the same trip but I set the report parameters to start from 7 and it is cut short, only showing the part of the trip that is after the start time. The api produces the same results. 

Additionally, if I set the end time to before 7:06 (when the trip ends) the trip would not show in the results at all.

I appreciate that these may be by design but I would have expected the start and end to have the same effect on the results.

Chris

[report-correct.xlsx](https://github.com/tananaev/traccar/files/1115671/report-correct.xlsx)
[report-incorrect.xlsx](https://github.com/tananaev/traccar/files/1115672/report-incorrect.xlsx)
